src/test/java/com/nike/cerberus/backstopper/apierror/contract/jsr303convention/ApplicationJsr303AnnotationTroller.java

```diff
@@ -1,64 +0,0 @@
-/*
```

Comment:
Why are these deleted?

Reply:
Because they are doing weird things with all the annotations despite claiming that they are only for JSR 308 Javax Validation annotations and it was causing issues. These tests were from the original boilerplate code and are now getting in the way.

We can make a todo to bring them back if you feel strongly.

Reply:
Was dep conflict, fixed and brought back tests.